pkgs/tools/package-management/nix-du/default.nix

```
@@ -1,13 +1,13 @@
 { stdenv, fetchFromGitHub, rustPlatform, nix, boost, graphviz, darwin }:
 rustPlatform.buildRustPackage rec {
   pname = "nix-du";
-  version = "0.3.0";
+  version = "0.3.0-git-master";
```

Since this no longer follows a release, please use the `unstable-YYYY-MM-DD` format of when the commit was applied
```suggestion
  version = "unstable-2019-07-15";
```

---

fixed.